tpot/base.py

```diff
@@ -929,6 +932,15 @@ def _save_periodic_pipeline(self):
         except Exception as e:
             self._update_pbar(pbar_num=0, pbar_msg='Failed saving periodic pipeline, exception:\n{}'.format(str(e)[:250]))
 
+    def _create_periodic_checkpoint_folder(self):
+        try:
+            os.makedirs(self.periodic_checkpoint_folder)
+        except OSError as e:
+            if e.errno == errno.EEXIST and os.path.isdir(self.periodic_checkpoint_folder):
+                pass
+            else:
+                raise
```

Comment:
I think  a ValueError with a message about this issue need be raised here

Reply:
This will now looks something like:

> Failed saving periodic pipeline, exception:
> Failed creating the periodic_checkpoint_folder:
> [WinError 5] Access is denied: './tmp'

As the error bubbles up. Is that ok?

Reply:
looks good! Thanks.